code/datums/gamemodes/mixed_rp.dm

```diff
@@ -2,8 +2,10 @@
 	name = "mixed (mild)"
 	config_tag = "mixed_rp"
 	latejoin_antag_compatible = 1
-	latejoin_antag_roles = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_WRESTLER, ROLE_ARCFIEND)
-	traitor_types = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_SPY_THIEF, ROLE_ARCFIEND)
+	 //went with a trivial solution of adding more identical items to the list
+	 //Input needed here
+	latejoin_antag_roles = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_WRESTLER, ROLE_ARCFIEND, ROLE_TRAITOR)
+	traitor_types = list(ROLE_TRAITOR, ROLE_CHANGELING, ROLE_VAMPIRE, ROLE_SPY_THIEF, ROLE_ARCFIEND, ROLE_TRAITOR)
```

Comment:
If you're going the route of adding a thing multiple times for higher chance, wouldn't it be more readable to have the same type next to each other, so you can see how often it is in the list more easily?

Reply:
Also, typo in the word "frequently" in changelog, I think?

Reply:
Hi! I chose to add them to the end as it's the least destructive change you'd do to a list. I really dont know how said array is used so I chose the method less prone to randomly break stuff somewhere else.

Typo also fixed.